src/api/payment-requests/payment-requests.md

```diff
@@ -561,6 +560,53 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 }
 {% endjson %}
 
+{% h4 Example response payload for a pre authorization Payment Request %}
+
+{% json %}
+{
+  "id": "MhocUmpxxmgdHjr7DgKoKw",
+  "shortCode": "CP-C7F-ZS5",
+  "url": "https://app.centrapay.com/pay/MhocUmpxxmgdHjr7DgKoKw",
```

Comment:
Where did this come from? It's not described in the model.

Please follow up with the author who wrote this and get it fixed in another PR.

Reply:
This field does exist and it is returned in our APIs however it is not documented in the model. [I have a seperate PR up that adds this field.](https://github.com/centrapay/centrapay-docs/pull/454)